openedx/core/djangoapps/user_authn/views/tests/test_reset_password.py

```diff
@@ -713,3 +713,87 @@ def test_reset_password_token_with_other_user(self):
 
         self.user = User.objects.get(pk=self.user.pk)
         self.assertFalse(self.user.is_active)
+
+
+@ddt.ddt
+@unittest.skipUnless(
+    settings.ROOT_URLCONF == "lms.urls",
+    "reset password tests should only run in LMS"
+)
+class ResetPasswordAPITest(CacheIsolationTestCase):
+    """Tests of the logistration API's password reset endpoint. """
+    request_factory = RequestFactory()
+    ENABLED_CACHES = ['default']
+
+    def setUp(self):
+        super(ResetPasswordAPITest, self).setUp()
+        self.user = UserFactory.create()
+        self.user.save()
+        self.token = default_token_generator.make_token(self.user)
+        self.uidb36 = int_to_base36(self.user.id)
+
+    @ddt.data(
+        (None, None, True),
+        (None, 'invalid_token', False),
+    )
+    @ddt.unpack
+    def test_invalid_token_in_password_reset_request(self, uidb36, token, status):
```

Comment:
Since this test case testing the both valid and invalid cases, we should rename it to `test_password_reset_request`. Also update the doc string.

Reply:
Any update?

Reply:
Changed it to `test_bad_token_in_password_reset_request`

Reply:
But this name still refers to; if we are only testing a bad token in this test but instead we are testing both bad and the correct one.

Reply:
Done